lib/Reader/CMakeLists.txt

```diff
@@ -21,3 +21,5 @@ add_llilcjit_library(LLILCReader
 if( NOT LLILC_BUILT_STANDALONE )
   add_dependencies(LLILCReader intrinsics_gen)
 endif()
+
+add_definitions(-DFEATURE_READYTORUN_COMPILER)
```

Comment:
We set `FEATURE_CORECLR` up in the root CMakeLists.txt. For consistency perhaps we should set `FEATURE_READYTORUN_COMPILER` up there too? Then you don't need to duplicate it in each leaf cmake file.